katamari/tools/build-clubhouse

```diff
@@ -41,6 +41,8 @@ DEFAULTS = {
     'clubhouse': {
         'run_lint': True,
         'run_tests': True,
+        'matomo_host': 'http://matomo.test.endlessos.org',
```

Comment:
`http://`... No secure communication?

Reply:
it's a test server, it'll be a secure one once we've the production one.